flow-state/server/rest/endpoints.go

```diff
@@ -56,6 +58,10 @@ func AppendEndpoints(router *httprouter.Router, logger log.Logger, exposeRecorde
 	router.GET("/v1/instances/:flowId/steps/status", sm.getStepsStatus)
 	router.GET("/v1/instances/:flowId/step/:stepId/taskdata", sm.getStepdataForActivity)
 	router.GET("/v1/flows", sm.getFlowNames)
+	router.GET("/v1/apps/:appName/versions", sm.getAppVersions)
+
+	router.GET("/v1/app/state/:app/:version", sm.getAppState)
+	router.POST("/v1/app/state/:app/:version/:toggle", sm.saveAppState)
```

Comment:
We should add delete method to remove toggle
router.POST("/v1/app/state/:app/:version", sm.enableAppState)
router.DELETE("/v1/app/state/:app/:version", sm.disableAppState)

Reply:
Added support for DELETE API with commit id ce1ac705b207a2551a136c8f9133fb113a652763